pkgs/top-level/ocaml-packages.nix

```diff
@@ -747,6 +752,40 @@ let
         inherit (pkgs) openssl;
       };
 
+    janeStreet_0_15 = (lib.makeScope self.newScope (self': with self'; {
+
+      janeStreet = import ../development/ocaml-modules/janestreet/0.15.nix {
+        self = self' // {
+          inherit (self) dune-configurator alcotest re num;
+          ppxlib = self.ppxlib.override { stdio = self'.stdio; };
+          janePackage = callPackage ../development/ocaml-modules/janestreet/janePackage_0_15.nix { };
+        };
+        inherit (pkgs) bash fetchpatch fzf lib openssl zstd;
+      };
+
+      # Packages that are not part of janestreet libraries, but still depend
+      # on v0.15 are kept in this scope, too.
+
+      bap = let
+        ppxlib = self.ppxlib.override { inherit stdio; };
+        lwt_ppx = self.lwt_ppx.override { inherit ppxlib; };
+        sedlex = self.sedlex.override { inherit ppxlib ppx_expect; };
+        in callPackage ../development/ocaml-modules/bap {
+          inherit (pkgs.llvmPackages) llvm;
+          ezjsonm = self.ezjsonm.override { inherit sexplib0; };
+          ppx_bitstring = self.ppx_bitstring.override { inherit ppxlib; };
+          ocurl = self.ocurl.override { inherit lwt_ppx; };
+          piqi = self.piqi.override { inherit sedlex; };
+          piqi-ocaml = self.piqi-ocaml.override { inherit piqi; };
+        };
+
+      ppx_bap = callPackage ../development/ocaml-modules/ppx_bap { };
+
+      torch = callPackage ../development/ocaml-modules/torch {
+        inherit (pkgs.python3Packages) torch;
+      };
```

Comment:
Please leave `torch` alone (see #249474).

Reply:
Ack, reverted changes to torch.